### ISSUE TITLE
Swift interop - nullability annotations

### ISSUE THREAD

**Issue by paweldudek** [PaddleHQ/Mac-Framework#183]
Hey there! First of all thank you for delivering a great SDK! 🎉 

There's one issue that I ran into when doing the integration, specifically incorrect nullability annotations for certain functions. 

Here're the ones that I found problematic: 

```
- (void)activateLicence:(nonnull NSString *)licenceCode email:(nonnull NSString *)email withCompletionBlock:(nonnull void (^)(BOOL activated, NSError * _Nonnull error))completionBlock;
```

The error passed in completion block is marked as `_Nonnull`, whereas if everything went fine it's a nil (hence should be `_Nullable`). 

Another one:

```
- (void)startPurchaseWithWindow:(nonnull NSWindow *)window completionBlock:(nullable void (^)( NSString * _Nullable email,  NSString * _Nullable licenceCode, BOOL activate, NSDictionary * _Nullable  checkoutData))completionBlock;
```

Documentation for this one states that 

>  * @param window An NSWindow the purchase window should be attached as a sheet to if required. Can be nil for this to presented modally..

so it either should be changed, or the nullability annotation should be changed to `nullable`. 

So far these are the only ones I found, but I only just started digging into this, I'll update this issue once I discover them. 

PS. I'd love to help out with this, but I see the SDK is closed source 😢

**Comment by GregularExpressions** [comment]
Hey @paweldudek thanks for the report! 

Your corrections make sense to me, I'll follow up with the engineering team and see if we can get these amended.

Thanks,
Greg.

**Comment by GregularExpressions** [comment]
Hey @paweldudek 

We believe we've corrected this in [v3.0.32](https://github.com/PaddleHQ/Mac-Framework/releases/tag/v3.0.32) 👍

Thanks,
Greg.
